src/util/interpolate-primitives.ts

```diff
@@ -0,0 +1,9 @@
+export function interpolateNumber(from: number, to: number, t: number): number {
```

Comment:
https://github.com/maplibre/maplibre-gl-js/blob/3d318e80220d083a960051e8601b107ce3f4ee77/src/util/util.ts#L210-L218

Reply:
Will try to bench the two lerps against each other at some point - for the fun of it :)